### ISSUE TITLE
can't install on iojs 1.1.0 - 1.2.0

### ISSUE THREAD

**Issue by oriSomething** [joliss/broccoli-sass#46]
os x 10.10.2
iojs 1.1.0 / 1.2.0 (via NVM)
npm 2.5.1

error in terminal:
```bash
npm ERR! node v1.2.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! node-sass@1.2.3 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@1.2.3 postinstall script 'node scripts/build.js'.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.
```

thanks!

**Comment by aexmachina** [comment]
This is a node-sass issue (it even says that in the error message). You should post it on node-sass.

**Comment by oriSomething** [comment]
thanks

**Comment by dschmidt** [comment]
I believe it could be fixed by updating node-sass to 2.0.0:
https://github.com/sass/node-sass/releases/tag/v2.0.0

**Comment by aexmachina** [comment]
Yep, I'm working on that... see #43.
